lib/node.ts

```diff
@@ -11,11 +12,13 @@ export {
   TabConsent,
 }
 
-export function attachToPage(page: any, url: string, rules: AutoCMP[], retries = 1) {
-  const frames: { [id: number]: any } = {};
+export async function attachToPage(page: any, url: string, rules: AutoCMP[], retries = 1) {
```

Comment:
Making this async is a breaking API change which will affect downstream users (i.e. [tracker-radar-collector](https://github.com/duckduckgo/tracker-radar-collector/blob/main/collectors/CMPCollector.js)).

Reply:
oh. Thanks for the heads up, I didn't realize it was part of an API